plugins/module_utils/network/ios/rm_templates/static_routes.py

```diff
@@ -38,7 +38,8 @@ def __init__(self, lines=None, module=None):
                 (\s(?P<dest>\S+))
                 (\s(?P<netmask>\S+))
                 (\s(?P<interface>(ACR|ATM-ACR|Analysis-Module|AppNav-Compress|AppNav-UnCompress|Async|Auto-Template|BD-VIF|BDI|BVI|Bluetooth|CDMA-Ix|CEM-ACR|CEM-PG|CTunnel|Container|Dialer|EsconPhy|Ethernet-Internal|Fcpa|Filter|Filtergroup|GigabitEthernet|TenGigabitEthernet|IMA-ACR|LongReachEthernet|Loopback|Lspvif|MFR|Multilink|NVI|Null|PROTECTION_GROUP|Port-channel|Portgroup|Pos-channel|SBC|SDH_ACR|SERIAL-ACR|SONET_ACR|SSLVPN-VIF|SYSCLOCK|Serial-PG|Service-Engine|TLS-VIF|Tunnel|VPN|Vif|Vir-cem-ACR|Virtual-PPP|Virtual-TokenRing)\S+))?
-                (\s(?P<forward_router_address>(?!multicast|dhcp|global|tag|track|permanent|name)\S+))?
+                # Had to add ip validation due to it picking up numbers as forward_router_address and negate just numbers
```

Comment:
got it, if we have to we have to! 
Let's not add comments in between parsers. I have faced issues with some versions if there were comments.
 You may add a comment to the PR incase someone is interested in the history.